scripts/copy-buildpacks.sh

```diff
@@ -2,7 +2,23 @@
 
 set -e
```

Comment:
Unless it breaks something (it shouldn't), you probably want `set -o pipefail` in this script now too as the script is literally using pipes now. :-)